zkevm-circuits/src/evm_circuit.rs

```diff
@@ -434,6 +438,8 @@ pub mod test {
             })
             .collect();
         let (active_gate_rows, active_lookup_rows) = EvmCircuit::get_active_rows(&block);
+        let mut block = block;
+        block.step_num_with_pad = ((1 << k) - 64) / STEP_HEIGHT;
```

Comment:
want to know why subtract  64 here.

Reply:
yes.   blinding rows is almost MAX(query of of each gate) + 3.   64 is bigger than it now. 

We 'd better avoid this hardcode value later.